fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/JDBCConstants.java

```diff
@@ -36,6 +36,14 @@
     public static final String DATE_END = "DATE_END";
     public static final String LATITUDE_VALUE = "LATITUDE_VALUE";
     public static final String LONGITUDE_VALUE = "LONGITUDE_VALUE";
+    public static final String RESOURCES = "_RESOURCES";
```

Comment:
I've been trying to keep the constant names identical to the string value for sanity. I don't really want to deviate from this. I think we should use a slightly different convention for prefix/postfix strings. Perhaps something as simple as _RESOURCES = "_RESOURCES"

Reply:
done - used your suggestion